docs/src/pages/components/autocomplete/GitHubLabel.tsx

```diff
@@ -62,41 +56,53 @@ const useStyles = makeStyles((theme: Theme) =>
       borderRadius: 2,
     },
     popper: {
-      border: '1px solid rgba(27,31,35,.15)',
```

Comment:
Migrating this demo to emotion is out of the scope of this PR. This will be done in another one.